examples/templates/exposed-thing/package.json

```diff
@@ -16,10 +16,10 @@
         "tslint": "5.12.1"
     },
     "dependencies": {
-        "@node-wot/binding-http": "0.8.3",
-        "@node-wot/core": "0.8.3",
+        "@node-wot/binding-http": "^0.8.2",
+        "@node-wot/core": "^0.8.2",
```

Comment:
This is correct. It might have been done (automatically) when upgrading the dependencies for the next release..

Reply:
@egekorkan I wonder whether we might stick to a *fix* version just to avoid any issues. Any thoughts?

Reply:
So we should definitely fix node-wot versions since node-wot does not follow the semantic versioning pattern. Only upgrades in the latest number should be allowed automatically, i.e. update from 0.8.2 to 0.8.3 should be always fine